ci-operator/step-registry/ovn/conf/vsphere-dualstack/ovn-conf-vsphere-dualstack-commands.sh

```diff
@@ -4,6 +4,7 @@ set -o nounset
 set -o pipefail
 
 echo "saving ipv4 vip configuration"
+curl -L https://github.com/mikefarah/yq/releases/latest/download/yq_linux_amd64 -o /tmp/yq && chmod +x /tmp/yq
```

Comment:
The download for `yq` would hit github bottleneck when there are concurrent running jobs, so suggest to move the binary install into image.

Reply:
Or maybe you can try `upi-installer` image for the step, the upi-installer already installed yq inside itself.

Reply:
`yq` version 3 or 4?  `upi-installer` has version `3`?

there is also

```
    name: cli-yq
    namespace: ocp
    tag: latest
```

Reply:
For now, the yq in upi-installer is on version 3, refer to https://github.com/openshift/installer/blob/master/images/installer/Dockerfile.upi.ci#L45.

Another enhancement PR for v4 is coming in https://github.com/openshift/installer/pull/7567

I have no idea about the yq version in cli-yq image. 

Seem like the step need both yq and jq command, so far, per my knowledge, only upi-installer can meet the requirement.

Reply:
`cli-yq` is the latest which is 4.x.  For `upi-installer` we would have to convert the `yq` commands to version 3 syntax, which is tricky.

https://github.com/openshift/release/blob/master/clusters/app.ci/supplemental-ci-images/cli-yq.yaml#L24

Reply:
if you prefer cli-yq, I am okay with it, can we also install jq there?

Reply:
@rbbratta, should I create a new image with the name 'cli-yq-jq,' or should I add the command to install jq in the existing 'cli-yq' image?

Reply:
I would add `jq` to `cli-yq`

Reply:
Added